README.md

```diff
@@ -529,6 +530,30 @@ OPTIONS
 
 _See code: [src/commands/es/indices/insert.ts](src/commands/es/indices/insert.ts)_
 
+## `kourou es:migrate`
+
+Migrate all the index from one Elasticsearch server to another
+
+```
+USAGE
+  $ kourou es:migrate
+
+OPTIONS
+  --batch-size=batch-size  [default: 1000] How many documents to move in batch per operation
```

Comment:
```suggestion
  --batch-size=batch-size  [default: 1000] The amount of documents to move in batch per operation
```

Reply:
I'll update it in the migrate.ts file and generate a new README